src/cas/pyodide-cas.ts

```diff
@@ -111,25 +111,87 @@ function usePythonConverter() {
     ['Sqrt', () => 'Sqrt'], // TODO: Replace with power
     ['Root', () => 'Root'], // TODO: Replace with power
     ['EqualEqual', () => 'sympy.Eq'],
+    ['Parentheses', () => ''],
+
+    ['Abs', () => 'sympy.Abs'],
+
+    ['Sin', () => 'sympy.sin'],
+    ['Cos', () => 'sympy.cos'],
+    ['Tan', () => 'sympy.tan'],
+
+    ['Sec', () => 'sympy.sec'],
+    ['Csc', () => 'sympy.csc'],
+    ['Cot', () => 'sympy.cot'],
+
+    ['Arcsin', () => 'sympy.asin'],
+    ['Arccos', () => 'sympy.acos'],
+    ['Arctan', () => 'sympy.atan'],
+
+    ['Asec', () => 'sympy.asec'],
+    ['Acsc', () => 'sympy.acsc'],
+    ['Atan', () => 'sympy.atan'],
+
+    ['Sinh', () => 'sympy.sinh'],
+    ['Cosh', () => 'sympy.cosh'],
+    ['Tanh', () => 'sympy.tanh'],
+
+    ['Sech', () => 'sympy.sech'],
+    ['Csch', () => 'sympy.csch'],
+    ['Coth', () => 'sympy.coth'],
+
+    ['Arsinh', () => 'sympy.asinh'],
+    ['Arcosh', () => 'sympy.acosh'],
+    ['Artanh', () => 'sympy.atanh'],
+
+    ['Asech', () => 'sympy.asech'],
+    ['Acsch', () => 'sympy.acsch'],
+    ['Acoth', () => 'sympy.acoth'],
+
+    ['Pi', () => 'sympy.pi'],
+    ['ImaginaryUnit', () => 'sympy.I'], // TODO: test
+    ['ExponentialE', () => 'sympy.E'], // TODO: test
+    ['GoldenRatio', () => 'sympy.GoldenRatio'], // TODO: test
+    ['EulerGamma', () => 'sympy.EulerGamma'], // TODO: test
+
+    // TODO: MathLive latex serial/parse destroys ln,log, etc? in ExpressionElement.vue line 78
+    // ['log', () => 'sympy.log'],
```

Comment:
I suppose this is another one of the current compute-engine limitations. Do you want to open an issue on the compute-engine repository or should I do it?

https://github.com/cortex-js/compute-engine/issues/13

Reply:
You seem to know more about how compute-engine works so I will let you open it. Please do notify me if you do so I can follow along.

Reply:
Okay, I added it as a comment to that compute-engine issue
https://github.com/cortex-js/compute-engine/issues/13#issuecomment-895831602

I could have opened a new issue, but I'm reasonably sure that the creator is already aware of this.